Dockerfile.base

```diff
@@ -9,11 +9,11 @@ EXPOSE 3000
 RUN mkdir -p /home/node/src
 WORKDIR /home/node/src
 
-RUN apk add --no-cache --virtual dumb-init-dependencies ca-certificates wget \
-	&& update-ca-certificates \
-	&& wget -O /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.2.0_amd64 \
-	&& chmod +x /usr/local/bin/dumb-init \
-	&& apk del dumb-init-dependencies
+RUN apk add --no-cache --virtual dumb-init-dependencies ca-certificates wget
```

Comment:
I've upgraded the script to do `docker build --squash`, which means chaining a massive command together is no longer necesarry